bookCopies/models.py

```diff
@@ -4,7 +4,16 @@
 
 
 # Create your models here.
-class bookCopies(models.Model):
+class BookCopy(models.Model):
     book = models.ForeignKey(Book, on_delete=models.CASCADE)
-    acquisition_date = models.DateField()
+    acquisition_date = models.DateField(
+        help_text='Please select date of acquisition of the book',
+        error_messages={
+            "empty": "date is required",
+            "invalid": "Please enter a valid date",
+        }
+    )
     condition = models.TextField()
+
+    class Meta:
```

Comment:
all meta classes should contain the "verbose name" and "verbose_plural_name" fields